CHANGELOG.md

```diff
@@ -6,14 +6,20 @@ adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Added
+
+- Added `:revision` configuration option which is automatically set to the current git revision by default
+
 ## [v0.11.0] - 2019-02-28
+
 When upgrading to v0.11, users should be aware of a few important changes:
 
 - You must be on Elixir 1.7+ and Erlang/OTP 21+
 - Due to the deprecation of `System.stacktrace/0` and the introduction of [`__STACKTRACE__`](https://hexdocs.pm/elixir/Kernel.SpecialForms.html#__STACKTRACE__/0), manually calling `Honeybadger.notify/3` will no longer include a stacktrace by default. See the [issue discussion](https://github.com/honeybadger-io/honeybadger-elixir/pull/204) for more details
-- A stacktrace can be manually provided from within a rescue/catch block via the __STACKTRACE__ macro, e.g. `Honeybadger.notify(SomeError, %{my: :context}, __STACKTRACE__)`
+- A stacktrace can be manually provided from within a rescue/catch block via the **STACKTRACE** macro, e.g. `Honeybadger.notify(SomeError, %{my: :context}, __STACKTRACE__)`
```

Comment:
`__STACKTRACE__` is a literal (the underscores are a part of the name). If this wasn't displaying properly, we could add the backticks around it to make it appear as code:

```suggestion
- A stacktrace can be manually provided from within a rescue/catch block via the `__STACKTRACE__` macro, e.g. `Honeybadger.notify(SomeError, %{my: :context}, __STACKTRACE__)`
```

Reply:
Of course no problem (regarding the whitespace)